test/core/get_local.wast

```diff
@@ -117,6 +117,26 @@
 )
 
 
+;; get_local should have retval
```

Comment:
I actually meant making the tests part of the "Invalid typing of access to parameters" section above, since they're really just additional variations of similar tests. You could even follow the naming scheme and name the functions i32-vs-empty etc.

Reply:
OK, done.